src/jira/client/jira-client.ts

```diff
@@ -261,7 +261,8 @@ export const getJiraClient = async (
 				const payload = {
 					builds: data.builds,
 					properties: {
-						gitHubInstallationId
+						gitHubInstallationId,
+						repositoryId
```

Comment:
Adding the `repositoryId` as a property when sending a build, so we can delete by that repositoryId later.